### PR TITLE
Create rclone.subfolder.conf.sample

### PR DESCRIPTION
**Pull request by STaRDoGG**
Based on the WebUI container for rClone, here: https://hub.docker.com/r/romancin/rclonebrowser

[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

### CODE REVIEW

File: rclone.subfolder.conf.sample [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    # enable the next two lines for ldap auth, also customize and enable ldap.conf in the default conf
+    #auth_request /auth;
+    #error_page 401 =200 /ldaplogin;
+
+    # enable for Authelia, also enable authelia-server.conf in the default site config
+    #include /config/nginx/authelia-location.conf;
+
+    include /config/nginx/proxy.conf;
+    resolver 127.0.0.11 valid=30s;
```
Comment (Roxedus) [inline rclone.subfolder.conf.sample:21]: Please update this to reflect the changes brought by #330

File: rclone.subfolder.conf.sample [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    set $upstream_port 5800;
+    set $upstream_proto http;
+    proxy_pass $upstream_proto://$upstream_app:$upstream_port;
+
+    rewrite /rclone(.*) $1 break;
+}
+
+ location ^~ /rclone/websockify {
+     include /config/nginx/proxy.conf;
+     resolver 127.0.0.11 valid=30s;
```
Comment (Roxedus) [inline rclone.subfolder.conf.sample:32]: Please update this to reflect the changes brought by #330

**Review by Roxedus — CHANGES_REQUESTED** [review]

**Comment by Roxedus** [comment]
Your latest push messed up all the PRs

**Comment by STaRDoGG** [comment]
> Your latest push messed up all the PRs

@Roxedus How?

All I did was click on Edit paste the line and save it. Admittedly, I'm roughly only a 4/10 as far as my GH skills go, because I rarely use it.

**Comment by Roxedus** [comment]
All your PRs became the petio subfolder 😅

**Comment by STaRDoGG** [comment]
> All your PRs became the petio subfolder 😅

@Roxedus Ughhh. No idea how tf that even happened. Save me some time and tell me the quickest/easist way to fix it? lmao.

I might end up just saying fuggit and letting someone else pr them. Ain nobody gat time fadis. lol

**Comment by STaRDoGG** [comment]
In retrospect I think I know what might've happened. I have a form recovery browser extension that may have gone kablooey and run amuck. Derp.

I'll revist all of my recent PR's and update with the previous code + the requested change in the coming week. Again, not being too good w/GH yet, I'd like to simply delete the previous mess and add the new ones, but I've also run into weird GH methods just trying that as well; like not being able to delete and having to "smush" it instead, or some weird ish like that. LOL.

**Review by Roxedus — APPROVED** [review]

**Pull request merged**